posthog/celery.py

```diff
@@ -40,6 +45,14 @@
 UPDATE_CACHED_DASHBOARD_ITEMS_INTERVAL_SECONDS = settings.UPDATE_CACHED_DASHBOARD_ITEMS_INTERVAL_SECONDS
 
 
+# following instructions from here https://django-structlog.readthedocs.io/en/latest/celery.html
+@setup_logging.connect
+def receiver_setup_logging(loglevel, logfile, format, colorize, **kwargs):  # pragma: no cover
+    # mypy thinks logging doesn't have `.config` but it does ¯\_(ツ)_/¯
+    logging.config.dictConfig(logs.LOGGING)  # type:ignore
+    logs.configure_structlog()
```

Comment:
non-blocking

`configure_structlog` will have already have been called just by importing logs right? Anyway sure it can't hurt, maybe celery does something nasty between import and this call.

Reply:
Well, with this call the format of the logs was the same between celery and django

_and_ 2 of the django test jobs consistently failed 🤷‍♀️

I'll merge without this in the hope I can get some output from my failing celery task... even if I need to revert afterwards :)